CommonMark/Formatters/HtmlFormatterSlim.cs

```diff
@@ -213,6 +213,104 @@ internal static void PrintPosition(HtmlTextWriter writer, Inline inline)
             writer.WriteConstant("\"");
         }
 
+        static void WriteTable(Block table, HtmlTextWriter writer, CommonMarkSettings settings, Stack<InlineStackEntry> stack)
+        {
+            if ((settings.AdditionalFeatures & CommonMarkAdditionalFeatures.GithubStyleTables) == 0)
+            {
+                throw new CommonMarkException("Table encountered in AST, but GithubStyleTables are not enabled");
```

Comment:
Is there a reason why the writer should throw the exception instead of just generating the HTML code? For example, if someone adds a different table style in the parser, the writer could work as is.